paymentsheet/src/main/java/com/stripe/android/link/account/LinkAccountManager.kt

```diff
@@ -28,6 +29,11 @@ internal interface LinkAccountManager {
      */
     val consumerPaymentDetails: StateFlow<ConsumerPaymentDetails?>
 
+    /**
+     * Cached shipping addresses for the current Link account.
+     */
+    var cachedShippingAddresses: ConsumerShippingAddresses?
```

Comment:
probably we want to rename `consumerPaymentDetails` to `cached` as well. Eventually we might also want to cache both in the `SavedStateHandle` to ensure it survives config changes, or ensure it gets re-fetched when lost.

Reply:
Yep, let's rethink how we store this state in the future!